dockerfiles/gw_docker_build_docs

```diff
@@ -0,0 +1,29 @@
+
+FROM ubuntu:20.04
+USER root
+
+RUN apt update -y && apt upgrade -y && \
+    apt install -y software-properties-common && \
+    add-apt-repository ppa:ubuntugis/ppa && \
+    apt update -y && apt install -y \
+    gdal-bin \
+    geotiff-bin \
+    git \
+    libgdal-dev \
+    libgl1 \
+    libspatialindex-dev \ 
+    wget \
+    python-is-python3 \
+    pip \
+    g++ 
+
+RUN pip install Cython numpy  
```

Comment:
Do we want/need to pin the versions on anything here?

Reply:
This build seems to be working. But I differ to you, I don't have a clear sense of how best to handle this.

Reply:
It might be a good idea for the Docker files.